packages/ozone/src/team/index.ts

```diff
@@ -211,13 +230,48 @@ export class TeamService {
     return profiles
   }
 
+  async syncMemberProfiles(appviewCreator: AppviewCreator): Promise<void> {
+    const { count } = await this.db.db
+      .selectFrom('member')
+      .select(this.db.db.fn.count<number>('did').as('count'))
+      .executeTakeFirstOrThrow()
+
+    let lastDid = ''
+    // Max 25 profiles can be fetched at a time so let's pull 25 members at a time from the db and update their profile details
+    for (let i = 0; i < count; i += 25) {
```

Comment:
Rather than fetching the `count` and relying on it not changing, you could alternately _only_ use the `lastDid` to iterate through.  I think it's sort of nice since you're already iterating by `lastDid`.  It would look more like:
```ts
let lastDid = ''
do {
  // ...
  lastDid = lastDid = dids.at(-1) || ''
} while (lastDid)
```